variables.keycloak.tf

```diff
@@ -41,4 +41,9 @@ variable "keycloak_user_app_role" {
 }
 variable "keycloak_add_identity_provider_azure" {
   type = bool
+}
+
+variable "deploy_automation_client" {
```

Comment:
I think, at the whole module level, the variable pattern is to prefix names with the "sub-module", so this one should probably be something like `keycloak_deploy_automation_client` (the other one is fine to me as it is already in the keycloak context)